kpconv_torch/io/xyz.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+
+def read_xyz(filepath):
+    """Takes a file path pointing on a 3D point .xyz (text) file and returns the points,
+    the associated colors and the associated classes.
+
+    Parameters
+    ----------
+    filepath: path to a 3D points file with .xyz
+
+    Returns
+    -------
+    points: 2D np.array with type float32
+    colors: 2D np.array with type uint8
+    labels: 1D np.array with type int32
+    """
+    data = np.loadtxt(filepath, delimiter=" ")
+    points = data[:, :3].astype(np.float32)
+    colors = data[:, 3:6].astype(np.uint8)
+    labels = np.squeeze(data[:, 6:7]).astype(np.int32)
+
+    return points, colors, labels
+
+
+def write_xyz(filepath, points, colors, labels):
+    """Creates a .xyz file from a 3D point cloud.
+    Parameters
+    ----------
+    filepath : pathlib.Path
+        Path where to save the data
+    points: 2D np.array with type float32 to save
+    colors: 2D np.array with type uint8 to save
+    labels: 1D np.array with type int32 to save
+    """
+    data = points
+    if colors is not None and 0 not in colors.shape:
```

Comment:
```suggestion
    if colors is not None and colors.shape[0] > 0: 
```

In my opinion this formulation is more meaningful.